### ISSUE TITLE
Game crashes after opening settings

### ISSUE THREAD

**Issue by dtluna** [OpenRA/OpenRA#11392]
The game crashes instantly after trying to open settings in any mod.

Exception log:
```
Tiberian Dawn Mod at Version release-20160508
on map 24ff645d468aa5a88898f6216cae19eaf2cab43e (shellmap by Chris Forbes).
Operating System: Linux (Unix 4.5.4.1)
Runtime Version: Mono 4.4.0 (Stable 4.4.0.40/f8474c4 Mon Mar 28 12:22:29 UTC 2016) CLR 4.0.30319.42000
Exception of type `System.Reflection.TargetInvocationException`: Exception has been thrown by the target of an invocation.
Inner Exception of type `System.NullReferenceException`: Object reference not set to an instance of an object
    at OpenRA.Platforms.Default.OpenAlSoundEngine.QueryDevices (System.String label, Int32 type) <0x406159d0 + 0x000a3> in <filename unknown>:0 
  at OpenRA.Platforms.Default.OpenAlSoundEngine.PhysicalDevices () <0x40615820 + 0x00033> in <filename unknown>:0 
  at OpenRA.Platforms.Default.OpenAlSoundEngine.AvailableDevices () <0x40615660 + 0x000ef> in <filename unknown>:0 
  at OpenRA.Sound.AvailableDevices () <0x40615450 + 0x0009c> in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.Logic.SettingsLogic.InitAudioPanel (OpenRA.Widgets.Widget panel) <0x40614500 + 0x00493> in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.Logic.SettingsLogic.RegisterSettingsPanel (PanelType type, System.Func`2 init, System.Func`2 reset, System.String panelID, System.String buttonID) <0x40600610 + 0x003ad> in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.Logic.SettingsLogic..ctor (OpenRA.Widgets.Widget widget, System.Action onExit, OpenRA.ModData modData, OpenRA.Graphics.WorldRenderer worldRenderer) <0x40610190 + 0x00537> in <filename unknown>:0 
  at (wrapper managed-to-native) System.Reflection.MonoCMethod:InternalInvoke (System.Reflection.MonoCMethod,object,object[],System.Exception&)
  at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) [0x00002] in /build/mono/src/mono-4.4.0/mcs/class/corlib/System.Reflection/MonoMethod.cs:644 
  at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) [0x00013] in /build/mono/src/mono-4.4.0/mcs/class/corlib/System.Reflection/MonoMethod.cs:650 
  at System.Reflection.MonoCMethod.DoInvoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00089] in /build/mono/src/mono-4.4.0/mcs/class/corlib/System.Reflection/MonoMethod.cs:635 
  at System.Reflection.MonoCMethod.Invoke (BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in /build/mono/src/mono-4.4.0/mcs/class/corlib/System.Reflection/MonoMethod.cs:663 
  at System.Reflection.ConstructorInfo.Invoke (System.Object[] parameters) [0x00000] in /build/mono/src/mono-4.4.0/mcs/class/corlib/System.Reflection/ConstructorInfo.cs:62 
  at OpenRA.ObjectCreator.CreateUsingArgs (System.Reflection.ConstructorInfo ctor, System.Collections.Generic.Dictionary`2 args) <0x4055f570 + 0x000e6> in <filename unknown>:0 
  at OpenRA.ObjectCreator.CreateObject[T] (System.String className, System.Collections.Generic.Dictionary`2 args) <0x40471f90 + 0x0010b> in <filename unknown>:0 
  at OpenRA.Widgets.Widget+<PostInit>c__AnonStorey0.<>m__0 (System.String l) <0x40561b90 + 0x00033> in <filename unknown>:0 
  at System.Linq.Enumerable+WhereSelectArrayIterator`2[TSource,TResult].MoveNext () [0x0004d] in /build/mono/src/mono-4.4.0/external/referencesource/System.Core/System/Linq/Enumerable.cs:327 
  at System.Linq.Buffer`1[TElement]..ctor (IEnumerable`1 source) [0x00083] in /build/mono/src/mono-4.4.0/external/referencesource/System.Core/System/Linq/Enumerable.cs:2568 
  at System.Linq.Enumerable.ToArray[TSource] (IEnumerable`1 source) [0x00011] in /build/mono/src/mono-4.4.0/external/referencesource/System.Core/System/Linq/Enumerable.cs:830 
  at OpenRA.Widgets.Widget.PostInit (OpenRA.Widgets.WidgetArgs args) <0x4055ed50 + 0x0014b> in <filename unknown>:0 
  at OpenRA.WidgetLoader.LoadWidget (OpenRA.Widgets.WidgetArgs args, OpenRA.Widgets.Widget parent, OpenRA.MiniYamlNode node) <0x4055c120 + 0x004f3> in <filename unknown>:0 
  at OpenRA.WidgetLoader.LoadWidget (OpenRA.Widgets.WidgetArgs args, OpenRA.Widgets.Widget parent, System.String w) <0x4055bff0 + 0x00067> in <filename unknown>:0 
  at OpenRA.Widgets.Ui.OpenWindow (System.String id, OpenRA.Widgets.WidgetArgs args) <0x405eaf20 + 0x00037> in <filename unknown>:0 
  at OpenRA.Game.OpenWindow (System.String widget, OpenRA.Widgets.WidgetArgs args) <0x4060f520 + 0x000d7> in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.Logic.MainMenuLogic+<MainMenuLogic>c__AnonStorey0.<>m__4 () <0x4060f3e0 + 0x0010b> in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.ButtonWidget.<ButtonWidget>m__A (MouseInput _) <0x405eaba0 + 0x0001f> in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.ButtonWidget.HandleMouseInput (MouseInput mi) <0x405e7100 + 0x0023d> in <filename unknown>:0 
  at OpenRA.Widgets.Widget.HandleMouseInputOuter (MouseInput mi) <0x405d7dd0 + 0x0026f> in <filename unknown>:0 
  at OpenRA.Widgets.Ui.HandleInput (MouseInput mi) <0x405d7c20 + 0x00097> in <filename unknown>:0 
  at OpenRA.DefaultInputHandler+<OnMouseInput>c__AnonStorey2.<>m__0 () <0x405d7bb0 + 0x00057> in <filename unknown>:0 
  at OpenRA.Sync.CheckSyncUnchanged[T] (OpenRA.World world, System.Func`1 fn) <0x405d0410 + 0x00097> in <filename unknown>:0 
  at OpenRA.DefaultInputHandler.OnMouseInput (MouseInput input) <0x405d7aa0 + 0x000f3> in <filename unknown>:0 
  at OpenRA.Platforms.Default.Sdl2Input.PumpInput (IInputHandler inputHandler) <0x4049a0f0 + 0x005e3> in <filename unknown>:0 
  at OpenRA.Platforms.Default.Sdl2GraphicsDevice.PumpInput (IInputHandler inputHandler) <0x4049a0b0 + 0x0002b> in <filename unknown>:0 
  at OpenRA.Renderer.EndFrame (IInputHandler inputHandler) <0x404982c0 + 0x00032> in <filename unknown>:0 
  at OpenRA.Game.RenderTick () <0x405d24a0 + 0x003bb> in <filename unknown>:0 
  at OpenRA.Game.Loop () <0x405ceb50 + 0x001e7> in <filename unknown>:0 
  at OpenRA.Game.Run () <0x405ce9e0 + 0x00077> in <filename unknown>:0 
  at OpenRA.Program.Run (System.String[] args) <0x403be7b0 + 0x0005f> in <filename unknown>:0 
  at OpenRA.Program.Main (System.String[] args) <0x403bdd60 + 0x000f7> in <filename unknown>:0 
```

**Comment by GraionDilach** [comment]
Duplicate of #11181.

**Comment by abcdefg30** [comment]
Looks like it, should be fixed by #11183 then.
